internal/executor/executor.go

```diff
@@ -16,38 +16,38 @@
 package executor
 
 import (
-	"bufio"
 	"fmt"
 	"io"
-	"log"
 	"os/exec"
 )
 
 type Executor interface {
 	Execute(output io.Writer, workingDir string, name string, args ...string) error
 	ExecuteAndCapture(output io.Writer, workingDir string, name string, args ...string) (string, error)
+	SetVerbose(bool)
 }
 
 type RealExecutor struct {
+	Verbose bool
 }
 
 func (e *RealExecutor) Execute(output io.Writer, workingDir string, name string, args ...string) error {
 	command := exec.Command(name, args...)
 	command.Dir = workingDir
-	tailer(output)(command.StdoutPipe())
-	tailer(output)(command.StderrPipe())
+	command.Stdout = output
```

Comment:
This used to cause a race condition (but not detected as there were no tests).
the previous go routines were not completed before the function returned.

By wiring the stdout and stderr to the file, the exec package handles this natively:
https://pkg.go.dev/os/exec#Cmd
```go
        // Stdout and Stderr specify the process's standard output and error.
	//[...]
	//
	// Otherwise, during the execution of the command a separate goroutine
	// reads from the process over a pipe and delivers that data to the
	// corresponding Writer. In this case, Wait does not complete until the
	// goroutine reaches EOF or encounters an error or a nonzero WaitDelay
	// expires.
	//
	// If Stdout and Stderr are the same writer, and have a type that can
	// be compared with ==, at most one goroutine at a time will call Write.
```